client/common/svg.download.js

```diff
@@ -112,3 +115,25 @@ export function downloadChart(mainGsel, svgName, styleParent = null) {
 
 	to_svg(svg, svgName, { apply_dom_styles: true })
 }
+
```

Comment:
I think these pdf-related functions should be in a separate file, `pdf.download.js`, to not affect the chunk size of the `svg.download.js` bundler. This way, other charts/plots that do not offer a pdf option do not have to trigger a larger chunk (808 KB) import. Later, can have `dom/downloadMenu.js` to offer different ways to download, and each menu option would dynamically import either svg.download, pdf.download, etc.